test-resources/bin/lib/constants

```diff
@@ -23,7 +23,7 @@ local_reg_port='5001'
 
 # defaults
 default_kind_cluster="dash0-operator-lab"
-default_kind_config="test-resources/kind-config-lr.yaml"
```

Comment:
I removed the old test-resources/kind-config.yaml and renamed the new test-resources/kind-config-lr.yaml to test-resources/kind-config.yaml.